subsys/dfu/dfu_target/src/dfu_target_suit.c

```diff
@@ -134,26 +170,66 @@ int dfu_target_suit_done(bool successful)
 
 int dfu_target_suit_schedule_update(int img_num)
 {
```

Comment:
Seems that behavior is different than expressed in header file

Reply:
Right, I forgot to change it. :)